### ISSUE TITLE
GKE Connection Test Failing

### ISSUE THREAD

**Issue by spiffyjr** [GoogleCloudPlatform/cloudbuild-integration-testing#50]
Is there another step required to get cloud build to talk to cluster nodes? In my own testing I was unable to get cloud build to talk to a node without whitelisting 0.0.0.0 and after running your setup I'm seeing the same results. 

```Step #9 - "integration tests": unable to connect. retrying in 3 sec...
Step #9 - "integration tests": url: http://35.188.172.115:30332
Step #9 - "integration tests": interval: 3
Step #9 - "integration tests": retries: 20
Step #9 - "integration tests": START CONNECTION TESTi
```

This never seems to complete.

**Comment by davidstanke** [comment]
Hey! I just merged @murog's revised instructions -- see the updated README and apply the Firewall changes. Let us know if it works. Thanks!

**Comment by murog** [comment]
fixed in #52
